package.json

```diff
@@ -115,8 +115,8 @@
     "generate-all-types-realm": "node ./scripts/generate-realm.js",
     "start": "electron .",
     "sentry:upload-symbols": "node ./scripts/sentry-symbols.js",
-    "test": "mocha-webpack --opts configs/mocha-webpack.opts",
-    "test:post-package": "mocha-webpack --opts configs/mocha-webpack.opts src/testing/post-packaging/index.ts",
+    "test": "npm run build && mocha-webpack --opts configs/mocha-webpack.opts",
+    "test:post-package": "npm run build && mocha-webpack --opts configs/mocha-webpack.opts src/testing/post-packaging/index.ts",
```

Comment:
I understand that it might be annoying to have to remember to build before testing (if you want the UI tests to run), but at the same time it isn't ideal to have to wait on the build to complete every time you run the tests, if you're iterating the tests and not the app code. Of cause running mocha in watch mode helps with that, but still.

Reply:
Ok.